### PR TITLE
Rewrite of the server in order to use graphql-tools

### PR DESCRIPTION
**Pull request by phiilu**
here you go :smile:

### CODE REVIEW

File: src/server/src/db.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const Sequelize = require('sequelize');
 const Fake = require('faker');
 
-const db = new Sequelize('postgres://postgres:root@localhost:5432/graphql');
+const db = new Sequelize('postgres://florian@localhost:5432/graphql');
```
Comment (zwenza) [inline src/server/src/db.js]: please revert 🙈

File: src/server/src/schema.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        users(id: ID, email: String): [User]
+    }
+
+    type Mutation {
+        addPerson(firstName: String!, lastName: String!, email: String!): User
+    }
+`;
+
+const schema = makeExecutableSchema({ typeDefs, resolvers });
+// addMockFunctionsToSchema({ schema });
```
Comment (zwenza) [inline src/server/src/schema.js]: do we need this?

**Comment by zwenza** [comment]
thanks for the PR! i added two comments 🙂

**Review by zwenza — CHANGES_REQUESTED** [review]

File: src/server/src/schema.js (hunk 2 above)
Comment (phiilu) [reply src/server/src/schema.js]: not really. it is useful if you don't already have data.

File: src/server/src/db.js (hunk 1 above)
Comment (phiilu) [reply src/server/src/db.js]: sure 👍

**Comment by phiilu** [comment]
I applied the requested changes 😄

**Review by zwenza — APPROVED** [review]

**Comment by zwenza** [comment]
thanks 💯

**Pull request merged**